integrationTests/ts/package.json

```diff
@@ -8,11 +8,6 @@
   "dependencies": {
     "graphql": "file:../graphql.tgz",
     "graphql-esm": "file:../graphql-esm.tgz",
-    "typescript-4.4": "npm:typescript@4.4.x",
-    "typescript-4.5": "npm:typescript@4.5.x",
-    "typescript-4.6": "npm:typescript@4.6.x",
-    "typescript-4.7": "npm:typescript@4.7.x",
-    "typescript-4.8": "npm:typescript@4.8.x",
```

Comment:
Regarding this, was already discussed at https://github.com/graphql/graphql-js/pull/4268#issuecomment-2445185032.